File: iniciante/1002/1002.cpp
#include<iostream>
#include <stdio.h>
#include <math.h>
#include <iomanip>

using namespace std;

int main(){
    double raio;
    cin >> raio;

    double area = 3.14159 * pow(raio, 2.0);
    
    cout << "A=" << setprecision(4) << fixed << area << endl;
}

File: iniciante/1002/1002.cs
using System; 

class URI {

    static void Main(string[] args) { 

        double pi = 3.14159;

        double raio = float.Parse(Console.ReadLine());

        double area = pi * Math.Pow(raio, 2);

        Console.WriteLine("A={0:F4}",area);
    }

}